exercises/food-chain/src/example/java/FoodChain.java

```diff
@@ -0,0 +1,77 @@
+import java.util.stream.IntStream;
+
+public class FoodChain {
+
+    private static final String[] ANIMALS = {
+        "fly",
+        "spider",
+        "bird",
+        "cat",
+        "dog",
+        "goat",
+        "cow"
+    };
+
+    private static final String[] FOLLOWUPS = {
+        "It wriggled and jiggled and tickled inside her.",
+        "How absurd to swallow a bird!",
+        "Imagine that, to swallow a cat!",
+        "What a hog, to swallow a dog!",
+        "Just opened her throat and swallowed a goat!",
+        "I don't know how she swallowed a cow!"
+    };
+
+    public String verse(int verse) {
+        return verseBeginning(verse) + verseMiddle(verse) + verseEnding(verse);
+    }
+
+    public String verse(int startVerse, int endVerse) {
```

Comment:
Maybe call this one `verses`? Since you've named your variables `startVerse` and `endVerse` it's very clear what it's doing though, so it's not very important :)